explainer/Explainer.md

```diff
@@ -150,5 +150,29 @@ private security data (tokens, passwords, etc).
 mechanisms to provide the necessary protection.  It does not define new ones.
 
 ## Implementations
+Implementations are described in more detail in the WoT Discovery Implementation Report,
+but in summary the following four implementations of the WoT Discovery TDD service
+has been performed:
+- LogiLabs TDD - An implementation supporting SPARQL queries, sponsored by Siemens.
+- WoT Hive TDD - An implementation supporting SPARQL queries, sponsored by Universidad Politécnica de Madrid.
+- TinyIoT TDD - An implementation supporting JSONPath queries, forked from an 
+  initial implmentation done by Fraunhofer as part of the LinkSmart project.
+- Minimal TDD - An implementation that intentionally includes only the core
+  features designed to support static implementations, such as devices statically
+  hosting multiple separate sensors.  Done by Intel.
+Except for the last implementation, all of these WoT TDD implementations also
+support registration and management of a dynamic database of TDs.
+
+There are other aspects of WoT Discovery that also need to be considered.
+We define a "Discoverer" as an entity that uses the WoT Discovery process to retrieve
+a TD.  We have a number of implementations of Discoverers, including integration into the 
+Eclipse node-wot project (an implementation of the informative 
```

Comment:
FYI: node-wot, at the moment, does not yet support discovery.
I think @JKRhb started work on discovery within his Dart implementation.

Reply:
Well, even just giving a URL pointing at a TD is considered "Discovery" - it's the "Direct" method.  Talked to Jan, and so far that's all he does (called "fetch").   However, security is a problem due to a lack of standard for setting up credentials, etc.  Also mapping credentials to id does not work when doing discovery, since you don't have the id yet.  This is also a security bootstrapping issue - which implementations like Dart and node-wot (and the Scripting API) need to be structured to support.

However... in the meantime, I will take node-wot out and add Dart, and Jan and I will talk about how to best support at least some aspects of Discovery in Dart in the short term.

Reply:
An example for the discovery implementation in `dart_wot` can be seen here: https://github.com/namib-project/dart_wot/blob/main/example/complex_example.dart#L129-L136 It is currently quite basic and does not support security schemes, but it reflects the current state of the Scripting API when it comes to discovery (at least for the direct method, the directory method is not supported yet).

Regarding node-wot, I meant that there is at least a `fetch` helper function defined (https://github.com/eclipse/thingweb.node-wot/blob/master/packages/core/src/helpers.ts#L152), which sort of acts like the direct method. So I think you could say that node-wot also qualifies as a discoverer. Not sure if this is enough to keep it in the explainer, though?

Reply:
I see Jans point... `direct` is supported in node-wot.